content/development-update/weekly-development-report-as-of-2025-05-23.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Weekly development report as of 2025-05-23
+tags:
+  - Weekly development updates
+  - Development
+  - Statistics
+url: ""
+image: https://ucarecdn.com/dd8e9208-05cb-4bcb-98b9-7f15821e8d30/
+image_text: Weekly development report as of 2025-05-23
+---
+
+### ECOSYSTEM UPDATES
+
+_The information in this section is based on publicly available sources and is provided for informational purposes only. It does not constitute an endorsement or recommendation of any project, product, or service mentioned._
+
+Cardano's ecosystem is experiencing steady growth in development, governance, and user activity. Currently, there are 1,999 projects in development. The number of delegated wallets has slightly increased to 1.33 million. Network usage is also up, with 109.73 million transactions processed. 
+
+Asset activity remains positive, with 10.81 million native tokens issued across 215,665 policies. Smart contract adoption is on the rise, featuring 134,180 Plutus scripts and 6,752 Aiken scripts. Governance participation has grown to 1,316 delegated representatives (DReps), with 957 of them active. Additionally, developer engagement has increased significantly, with 391 commits on GitHub.
+
+In other news:
+
+*   Flow DeFi has released their [litepaper](https://x.com/flowdefi/status/1925257221530640520) and announced an integration with [Strike Finance](https://x.com/flowdefi/status/1924500743567839489).
+    
+*   [Trust Wallet](https://x.com/TrustWallet/status/1921930417336414348) has rolled out an update enabling in-app DRep delegation for Cardano users.
+    
+*   [Emurgo](https://x.com/bigpeyYT/status/1924475894514553287) has confirmed that it will no longer participate in Cardano governance using Genesis ADA.
```

Comment:
```suggestion
*   [Emurgo](https://x.com/bigpeyYT/status/1924475894514553287) has confirmed that it will no longer participate in Cardano governance using genesis ada.
```